inst/emails/contact_annual_report

```diff
@@ -1,12 +1,14 @@
+The NSF-funded Arctic Data Center (https://arcticdata.io/) support team would like to remind you that the deadline to submit your annual report is approaching. 
+ 
+NSF requires that annual reports be submitted at least 90 days before the "anniversary" of your award, and that you must submit applicable data to the Arctic Data Center or another suitable archive and include the DOI or another identifier of the data in your report. See the reporting guidelines{https://www.nsf.gov/pubs/2016/nsf16055/nsf16055.jsp} from NSF for details, but generally NSF requires that URL's for archived metadata and data should be included in these reports in the section entitled "Products-Websites".
 
- If you have not already done so, please submit your applicable data and metadata by going to https://arcticdata.io/catalog/#share. You can also visit our data submission guidelines page (https://arcticdata.io/submit/) for more information about the Arctic Data Center submission process.
+If you have not already done so, please submit your applicable data and metadata by going to https://arcticdata.io/catalog/#share. You can also visit our data submission guidelines page (https://arcticdata.io/submit/) for more information about the submission process.
 
- Please be sure to submit your data and metadata to the Arctic Data Center sufficiently before your report is due to allow adequate processing time for us to review your submission and for any needed revisions to be prepared.  While we strive to process data as quickly as possible, larger data sets can take significantly more time to process, so please plan accordingly.  Also, there will be less need for multiple rounds of revisions if your initial submission uses open, non-proprietary data formats, includes data that is well-organized, and includes high quality, detailed documentation.  For example, you should document all aspects of your data, including detailed methodological descriptions and structural information about the contents of your data.  See our data submission guidelines page (https://arcticdata.io/submit/) for recommendations.
+Prior to issuing a DOI, there is a processing period for the Arctic Data Centre to review a submission. While we strive to process data as quickly as possible, larger data sets can take significantly more time to process. We advise that this is taken to account when planning a submission to the Arctic Data Center for your report.
 
- If you have any questions about this process please reply to this email and we will respond as soon as possible.
+If you have any questions about any of the above, don't hestitate to email us and we will respond as soon as possible.
```

Comment:
I think the reason the prior text said "respond to this email"  rather than "email us" is to keep their response tied to the RT ticket created for their award.  If you still want to change it to "email us", we should provide the `support@arcticdata.io` email address.